src/test/java/org/dataloader/fixtures/TestKit.java

```diff
@@ -61,43 +60,14 @@ public static <K, V> BatchLoader<K, V> keysAsValues(List<List<K>> loadCalls) {
         };
     }
 
-    public static <K, V> BatchLoader<K, V> keysAsValuesAsync(Duration delay) {
```

Comment:
Figured I'd clean up the now unused methods - we should be using the `TestDataLoaderFactory`-provided `DataLoader`s anyway, if we feel the need to reach out for these older ones.